<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RConsoleDialog</class>
 <widget class="QDialog" name="RConsoleDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="3">
       <widget class="QPushButton" name="runButton">
        <property name="text">
         <string>Run Script</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear Script</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QTextEdit" name="scriptEdit">
        <property name="acceptRichText">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>R Script</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>scriptEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
